examples/durable/FunctionChainingApp/MyOrchestrator/run.ps1

```diff
@@ -0,0 +1,15 @@
+using namespace System.Net
+
+param($Context)
+
+Write-Host "MyOrchestrator: started. Input: $($Context.Input)"
+
+$output = @()
```

Comment:
Using this pattern ```$output = @()```, each time you add a new item ``` $output +=```, the whole array is copied, which is not very efficient. Please use ```$output = New-Object -TypeName System.Collections.Generic.List[System.Object]``` instead.

Reply:
This is a sample demonstrating durable functions usage, so it is optimized for readability. I wouldn't want the reader to think "oh, do I have to use `New-Object -TypeName System.Collections.Generic.List[System.Object]` to make my durable function work?" Comparing to the overhead of reading, thinking, asking, and answering, the performance overhead is going to be negligible in most cases.

Reply:
I agree for the Durable usage. However, in general, we should discourage users from using pattern. Resolving conversation.